src/poker/game/GamePlay.java

```diff
@@ -0,0 +1,362 @@
+        System.out.println(playerSmallBlind.getHandPlayer());
+        System.out.println(playerSmallBlind.getStackPlayer());
+        updatePotAndStackAmount(gameState.bigBlind, playerBigBlind);
+        updatePotAndStackAmount(gameState.smallBlind, playerSmallBlind);
+        playerSmallBlind.increasePlayerMoneyAtStrit(gameState.smallBlind);
+        playerBigBlind.increasePlayerMoneyAtStrit(gameState.bigBlind);
+        System.out.println("Zaczyna " + playerSmallBlind.getNamePlayer());
+        System.out.println(playerBigBlind.getNamePlayer() + " bigBlind " + gameState.bigBlind + "$");
+        System.out.println(playerSmallBlind.getNamePlayer() + " smallBlind " + gameState.smallBlind + "$");
+        printPot();
+        System.out.println(playerSmallBlind.getNamePlayer() + " podejmuje decyzje");
+        System.out.println(playerSmallBlind.getNamePlayer() + " " + amountToCall(playerBigBlind, playerSmallBlind) + "$ to call");
+        // System.out.println("1 = call 2 = raise");
+    }
+
+    public void printPot() {
+        System.out.println("Pot wynosi " + gameState.getPot() + "$");
+    }
+
+    public void layCards(GameState gameState) {
+        boolean isDraw = false;
+        if (gameState.getGamePhase() == GamePhase.PREFLOP) {
+            printPot();
+            System.out.println("Flop");
+            printBoard(gameState.getGamePhase());
+        } else if (gameState.getGamePhase() == GamePhase.FLOP) {
+            printPot();
+            System.out.println("Turn");
+            printBoard(gameState.getGamePhase());
+        } else if (gameState.getGamePhase() == GamePhase.TURN) {
+            printPot();
+            System.out.println("River");
+            printBoard(gameState.getGamePhase());
+        } else {
+            GameResult.result whoWins = equityEvaluator.updatePlayerWins(result.getBoard(), result.getHand1(), result.getHand2());
+            System.out.print(whoWins + " " + gameState.getPot() + "$");
+            if (whoWins == GameResult.result.DRAW) {
+                isDraw = true;
+            }
+            if (whoWins == GameResult.result.WIN_PLAYER1) {
+                updatePlayersOnShowdownResult(player1, player2, isDraw);
+            } else {
+                updatePlayersOnShowdownResult(player2, player1, isDraw);
+            }
+        }
+    }
+
+    public void printBoard(GamePhase gamePhase) {
+        int endIndex = BOARD_PRINT_INDEXES.get(gamePhase);
+        for (int i = 0; i < endIndex; i++) {
+            System.out.print(result.getBoard().get(i) + " ");
+        }
+        System.out.println();
+    }
+
+    public void handleRaisePlayer(Player activePlayer, Player nonActivePlayer) {
+        System.out.println("Podaj do ilu, raise minimum " + Math.max(2, 2 *  nonActivePlayer.getPreviousRaiseOrBet()));
+        double raise = 0;
+        Scanner scanner = new Scanner(System.in);
+        raise = scanner.nextDouble();
+        if (raise >= activePlayer.stack) {
```

Comment:
dorob w jakims dobrym miejscu funckje handleAllIn() i tam daj cala logike jak jest all in